code/__defines/mobs.dm

```diff
@@ -205,10 +205,14 @@
 #define AUGMENTATION_ORGANIC  2
 
 // Prosthetic helpers.
-#define BP_IS_PROSTHETIC(org) (!QDELETED(org) && (org.status & ORGAN_PROSTHETIC))
-#define BP_IS_ASSISTED(org)   (!QDELETED(org) && (org.status & ORGAN_ASSISTED))
-#define BP_IS_BRITTLE(org)    (!QDELETED(org) && (org.status & ORGAN_BRITTLE))
-#define BP_IS_CRYSTAL(org)    (!QDELETED(org) && (org.status & ORGAN_CRYSTAL))
+#define BP_IS_PROSTHETIC(org) (!QDELETED(org) && (org.organ_properties & ORGAN_PROP_PROSTHETIC))
+#define BP_IS_BRITTLE(org)    (!QDELETED(org) && (org.status           & ORGAN_BRITTLE))
+#define BP_IS_CRYSTAL(org)    (!QDELETED(org) && (org.organ_properties & ORGAN_PROP_CRYSTAL))
+
+//Prosthetics setter
+//Since assisted and prosthetic are mutually exclusive set them using this helper for convenience
```

Comment:
Comment is now out of date ftr

Reply:
I wonder if the setters are actually needed anymore though. I only made them because assisted / prosthetic are exclusive. But with assisted gone, its not doing much.